src/main/scala/tusday.scala

```diff
@@ -0,0 +1,58 @@
+object tusday extends App{
+
+  val x = 99 to 0 by -1
+
+  def song(counter: Int):String = {
+    val counter2 = counter-1
+    s"""         |   $counter bottles of beer on the wall
+                 |   $counter bottles of beer
+                 |  take one down and pass it around, $counter2 bottles of beer on the wall """
+  }
+
+  x.foreach(bigint => if(bigint >= 1)(println(song(bigint)))else(print("No more bottles of beer on the wall, no more bottles of beer. Go to the store and buy some more, 99 bottles of beer on the wall")))
+
+
+
+  def recursionSong(counter:Int):Unit={
+    if(counter> 1)
+      {
+        println(s"""|   $counter bottles of beer on the wall
+                   |   $counter bottles of beer
+                   |  take one down and pass it around, ${counter-1} bottles of beer on the wall """)
+        recursionSong(counter-1)
+      }
+    else if(counter ==1)
+    {
+      println(s"""|   $counter bottles of beer on the wall
+                  |   $counter bottles of beer
+                  |  take one down and pass it around, no more bottles of beer on the wall """)
+      recursionSong(counter-1)
+    }
+    else if(counter ==0)
```

Comment:
pretty sure it would still work if this was just a simple else rather than else if